src/util/is-mobile.ts

```diff
@@ -0,0 +1,5 @@
+const isMobile = () => {
+  return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
```

Comment:
```suggestion
  return (navigator as any).userAgentData?.mobile ?? /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
```
for browsers that support it (probably more reliable), fallback to your way